Library/Homebrew/cmd/update-report.rb

```diff
@@ -866,7 +870,12 @@ def dump_deleted_formula_report(report_all)
       end
     end.compact
 
-    output_dump_formula_or_cask_report "Deleted Formulae", formulae
+    title = if report_all
+      "Deleted Formulae"
+    else
+      "Deleted and Installed Formulae"
```

Comment:
@EricFromCanada as one of our best prose people (and welcome thoughts from any other maintainers who read this!): does this seem like good wording to you? If not: any other suggestions?

Reply:
Feels like there could be something better since it took me a couple seconds to understand what this meant but I don't have anything to suggest.

Reply:
At least to me, the use of "and" (in "Deleted and Installed") makes it sound like they're two separate categories. "Deleted Installed Formulae" or "Deleted Formulae (Installed)" _may_ be a little closer to what we're trying to convey.

Reply:
"Deleted Installed Formulae" works for me, thanks @samford.